tech_specs/component_definitions.md

```diff
@@ -25,24 +25,32 @@ In order to make these ends meet, we can identify the following levels:
 
 ### Control Center (eg. "Brewery Two")
 
-The top level grouping inside the GUI application. All sensors, actuators, and settings for a single service are somewhere in here. UI blocks are stored here, so multiple processes can share them.
+The top level grouping inside the GUI application. All sensors, actuators, and settings for a single service are somewhere in here. UI blocks are stored here, so multiple views can share them.
 
-### Dashboard (eg. "Fermentation" / "Overview" / "All temperature sensors")
+### View (eg. "Fermentation" / "Overview" / "All temperature sensors")
 
-A clearly defined subset of an activity. This involves a subset of all UI blocks, but generally uses all assigned UI blocks simultaneously.
+A canvas displaying one or more widgets. Views do not directly display data or actions: they contain all objects (widgets) that do.
 
-It should be possible that multiple dashboards share physical UI blocks.
+Views can contain other views. Views can share widgets.
```

Comment:
Not sure about this. It will probably be easier if we define widgets inside a view.
If 2 views want to display the same widget, it is defined in each. They map to the same hardware/service data, but this is separate from the view widget.